s4cmd.py

```diff
@@ -38,8 +38,11 @@
   def cmp(a, b):
     return (a > b) - (a < b)
 
-from functools import cmp_to_key
-
+if sys.version_info < (2, 7):
+    # Python < 2.7 doesn't have the cmp_to_key function.
```

Comment:
We use 2 spaces instead of 4 for all indent. Do you mind if you can fix all the indentation?

Thanks for doing this.